etc/ReplayOfflineConfiguration.py

```diff
@@ -796,7 +818,10 @@
 ### special AlcaRaw PDs ###
 ###########################
 
-DATASETS = ["AlCaLumiPixels"]
+DATASETS = ["AlCaLumiPixels", "AlCaLumiPixels0", "AlCaLumiPixels1", "AlCaLumiPixels2", "AlCaLumiPixels3",
+            "AlCaLumiPixels4", "AlCaLumiPixels5", "AlCaLumiPixels6", "AlCaLumiPixels7",
+            "AlCaLumiPixels8", "AlCaLumiPixels9", "AlCaLumiPixels10", "AlCaLumiPixels11",
+            "AlCaLumiPixels12"]
```

Comment:
Same comment: the dataset `AlCaLumiPixels` has been replaced  by `AlCaLumiPixelCountsPrompt`, does it make sense to keep this?